calico/felix/endpoint.py

```diff
@@ -339,37 +362,43 @@ def _remove_chains(self):
             _log.exception("Failed to delete chains for %s", self)
             self._failed = True
 
-    def _configure_interface(self):
+    def _configure_interface(self, mac_changed):
         """
         Applies sysctls and routes to the interface.
+
+        :param: bool mac_changed: Has the MAC address changed since it was last
+        configured? If so, we reconfigure ARP for the interface.
```

Comment:
This comment should specify IPv4 (I'd quite like to see an explanation of why we can't reset ARP for IPv6 too).